distribution/docs/pom.xml

```diff
@@ -63,6 +63,7 @@
             <plugin>
                 <groupId>org.owasp</groupId>
                 <artifactId>dependency-check-maven</artifactId>
+                <version>1.2.9</version>
```

Comment:
Double checking on that. I may have added it will tracking down a build issue and forgotten to take it back out.